lib/stat_tracker.rb

```diff
@@ -6,18 +6,38 @@
 require './lib/team_info'
 
 class StatTracker
-  attr_reader :games
+  attr_reader :games, :teams, :game_team_stats
 
-  @games = []
-  @teams = []
-  @team_info_path = './data/team_info.csv'
+  def initialize(locations)
+    @games = []
+    @teams = []
+    @game_team_stats = []
+    get_game_info(locations[:games])
+    get_team_info(locations[:teams])
+    get_game_teams_stats(locations[:game_teams])
+  end
```

Comment:
Adding the get_* methods as part of the initialize was the missing key to making this work.